docs/modules/integrations/pages/xray-exporter.adoc

```diff
@@ -46,6 +48,10 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
 |false
 |The key of a `Test Set` to which the exported test cases will be added
 
+|`xray-exporter.jira-instance-key`
+|false
+|The key of previousle configured JIRA instance, in case of missing value it will be evaluated automatically based on issue keys being exported
```

Comment:
```suggestion
|The key of the configured JIRA instance, in case of missing value it will be evaluated automatically based on issue keys being exported
```